api/models/v0.2/modelResponse.js

```diff
@@ -76,14 +68,12 @@ function parseTypeURL(str) {
  */
 function modelErrorResponseData(obj) {
   return {
-    data: {
```

Comment:
We are intentionally removing the containing array so that we only return one patron record.